pkg/agent/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -289,10 +297,19 @@ func (c *Controller) Run(stopCh <-chan struct{}) error {
 	// Use NonSlidingUntil so that normal reconnection (disconnected after
 	// running a while) can reconnect immediately while abnormal reconnection
 	// won't be too aggressive.
+
```

Comment:
The above comment is for the following code, please don't add the empty line if no strong reason.

Reply:
Restored